app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> None:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            Cafe.visit_cafe(self=cafe, visitor=friend)
```

Comment:
Kate, why did you call visit_cafe method with class object and then put the instance with self key. Its up to you, just interesting =)

Reply:
no , it's not up to you in this case. The correct way is `cafe.visit_cafe(friend)`